src/main/java/io/quarkus/gizmo2/impl/ConstructorDescImpl.java

```diff
@@ -18,7 +18,7 @@ public ConstructorDescImpl(final ClassDesc owner, final MethodTypeDesc type) {
         }
         this.owner = owner;
         this.type = type;
-        hashCode = Objects.hash(owner, "<init>", type);
+        hashCode = buildHashCode();
```

Comment:
This one I don't quite understand; we were already caching the hash code, so what is the difference here?